### ISSUE TITLE
Module dependency issue with puppetlabs-puppetdb >= 3.0.0

### ISSUE THREAD

**Issue by alexanderfahlke** [razorsedge/puppet-cloudera#11]
The module dependencies are conflicting with the postgresql module.
 
puppetlabs-puppetdb depends on ```puppetlabs/postgresql (>= 3.1.0 <4.0.0)```
puppet-cloudera depends on ```puppetlabs/postgresql (>=2.1.0 <3.0.0)```

Could you please update the postgresql dependency?

Installing puppet-cloudera by ignoring the dependencies and manually add the modules is a workaround but a very ugly one and will mark puppetlabs-postgresql as invalid.

```
$ puppet module install puppetlabs-puppetdb
$ puppet module install --ignore-dependencies razorsedge-cloudera

$ puppet module install darin-zypprepo
$ puppet module install domcleal-augeasproviders
$ puppet module install nanliu-staging
$ puppet module install puppetlabs-java_ks
$ puppet module install stahnma-epel
$ puppet module install puppetlabs-mysql
```

```
$ puppet module list
Warning: Module 'puppetlabs-postgresql' (v3.3.3) fails to meet some dependencies:
  'razorsedge-cloudera' (v2.0.2) requires 'puppetlabs-postgresql' (>=2.1.0 <3.0.0)
/etc/puppet/modules
├── darin-zypprepo (v1.0.1)
├── domcleal-augeasproviders (v1.1.0)
├── nanliu-staging (v0.4.0)
├── puppetlabs-apt (v1.4.2)
├── puppetlabs-concat (v1.1.0)
├── puppetlabs-firewall (v1.1.1)
├── puppetlabs-inifile (v1.0.3)
├── puppetlabs-java_ks (v1.2.3)
├── puppetlabs-mysql (v2.2.3)
├── puppetlabs-postgresql (v3.3.3)  invalid
├── puppetlabs-puppetdb (v3.0.1)
├── puppetlabs-stdlib (v4.2.1)
├── razorsedge-cloudera (v2.0.2)
└── stahnma-epel (v0.0.6)
```

Note: I am not using postgresql as database.

**Comment by razorsedge** [comment]
Fixed in commit 1763ccc4169c4b84b0ef13e19bc4742bb3f01d71.

**Comment by razorsedge** [comment]
This is fixed in version 3.0.0 on the [Forge](https://forge.puppetlabs.com/razorsedge/cloudera).
